languages/javascript/exercises/concept/array-analysis/.docs/hints.md

```diff
@@ -0,0 +1,31 @@
+### 1. Check card position
+
+- Array indices starts at `0`.
+- There is a [built-in][indexof_method_docs] method to get given value index.
+
+### 2. Check whether the given card is in the stack
+
+- There is a [built-in][includes_method_docs] method to check whether the value is present in the array
+
+### 3. Check whether every card in stack is even number
+
+- There is a [built-in][every_method_docs] method to check if each value passes a predicate
```

Comment:
I'm wondering if students understand what a _predicate_ is. I'm guessing some of them don't. Maybe rephrase a bit, introduce the term in the introduction or link to a document explaining what a predicate is?

Reply:
I don't think we should rephrase. I do think we _could_ explain it again in `introduction.md`, BUT this exercise relies on `callbacks` as a Concept, which should explain this.

Reply:
(I'll just write a small blog post on this and put it up -- how's that?)

Reply:
https://derk-jan.com/2020/05/predicate/